src/main/java/com/jcabi/github/User.java

```diff
@@ -90,6 +102,25 @@
     UserEmails emails();
 
     /**
+     * Returns all notifications of a user. Wraps the call "List your
```

Comment:
Just one sentence in the first line of the method.
All additional explanations should be in the following lines, I'm sure that single space indentation shouldn't be required if you do it this way.